Makefile

```diff
@@ -2,6 +2,7 @@
 
 WEB_FOLDER := web
 PORT := 7777
+DOCKER_IP := $(shell docker-machine ip default)
```

Comment:
Is this gonna work if you don't use `docker-machine`?

Reply:
Not sure to be honest. I was just trying to provide a more "sane default". I don't really know how others work when using compose and tried with what I usually have. 

I'll try some different things and will adapt to a sane default.

Reply:
yeah, this works for mac, but in Linux docker-machine is not necessary as there is no underlying virtual machine.  I believe for most mac users (not using [mobymac](https://github.com/dziemba/mobymac but docker for mac instead) `localhost` could be a sane default

Reply:
I'm considering just supporting and `.env` file and letting people change it locally defaulting to `localhost`

Reply:
@marctc @jcreixell this has been addressed.
Now it defaults to `localhost` unless specified something different.